keps/sig-network/1880-multiple-service-cidrs/README.md

```diff
@@ -309,68 +308,25 @@ All the apiservers will be synchronized on the ServiceCIDR and default Service c
 Changes on the configuration imply manual removal of the ServiceCIDR and default Service, automatically
 the rest of the apiservers will race and the winner will set the configuration of the cluster.
 
-Pros:
-- Simple to implement
-- Align with current behavior of kubernetes.default, though this can be a Con as well, since this
-the existing behavior was unexpected
-Cons:
-- Requires manual intervention
-
-Option 2:
-... with a special label `networking.kubernetes.io/service-cidr-from-flags` set to `"true"`.
-
-It now has to handle the possibility of multiple ServiceCIDR with the special label, and
-also updating the configuration, per example, from single-stack to dual-stack.
-
-The new bootstrap process will be:
+This behavior align with current behavior of kubernetes.default, that it makes it consistent and easier to think
```

Comment:
The code above says "if ServiceCIDR `kubernetes` does not exist" - will we also check that it has the correct value?  E.g. if I user were to very quickly delete, clear finalizer, and recreate with a bad value - would we fix it?

Reply:
Also, if multiple API servers have different flags, they will just fight - is that OK?

Reply:
I see some on this below.  I think I would a) add a bit more detail above; b) add here something like: "this loop runs independently in each apiserver, if apiservers have different flags, then ..." and sepll out what we expect to happen.

I think I'd also use the example below, where you change the IP from 10. to 192., and show the intermediate state where one apiserver is old config and one is new, and what we expect to happen in that state (specifically we could stay in that state for hours or days).